tests/transforms/test_broadcast_expand.py

```diff
@@ -174,54 +151,52 @@ def test_not_copied(self):
 
     @pytest.mark.autograd
     @pytest.mark.filterwarnings("ignore:Output seems independent of input")
-    @pytest.mark.parametrize("params, size", parameters_and_size)
+    @pytest.mark.parametrize("params", parameters)
     @pytest.mark.parametrize("obs, exp_fn", observables_and_exp_fns)
-    def test_autograd(self, params, size, obs, exp_fn):
+    @pytest.mark.parametrize("diff_method", ["parameter-shift", "backprop"])
```

Comment:
Added `diff_method` parametrization to check that Autograd + broadcasting expansion works and is diffable.